src/config/env.py

```diff
@@ -0,0 +1,105 @@
+"""
+env.py
```

Comment:
The first line should be a single-line top-level description of the module (preferably < 80 chars). See: https://google.github.io/styleguide/pyguide.html#382-modules

We're not too strict, but trying to get better :sweat_smile: